.github/workflows/test.yml

```diff
@@ -37,17 +38,18 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-13, macos-14]
-        python_version: ['3.12']
+        python_version: ['3.13']
         include:
         - os: ubuntu-latest
-          python_version: '3.8'
+          python_version: '3.11'
     timeout-minutes: 180
     steps:
     - uses: actions/checkout@v4
     - uses: actions/setup-python@v5
       name: Install Python ${{ matrix.python_version }}
       with:
         python-version: ${{ matrix.python_version }}
+        allow-prereleases: true
```

Comment:
```suggestion
```
To be removed once 3.13 reaches GA

Reply:
per https://github.com/pypa/cibuildwheel/pull/1912#issuecomment-2400064916, just keep it, it's indeed harmless